<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
          "http://www.w3.org/TR/html1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

  <head>
    <title> Software </title>
    <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
    <link rel="stylesheet" type="text/css" href="../iystyle2.css" />

  </head>

    <h1> Software </h1>
<hr size = "7" noshade="noshade" />



<table noborder width="100%" cellspacing="50" cellpadding="0">
  <tr> <td valign="top" align="left">

<dl>
<p>
  <dt><a href="https://github.com/de-Boer-Lab/random-promoter-dream-challenge-2022">Python/PrixFixe</a></dt>
  <dd> A Python module prepared for <a href="https://www.synapse.org/#!Synapse:syn28469146/wiki/617075">Random Promoter Dream Challenge 2022</a>. Sequence-to-expression models trained on expression measurements of random promoter sequences. The models would receive regulatory DNA sequence as input and use it to predict the corresponding gene expression value. Our team placed third in the competition and collaborated with other top teams to provide code for the development of PrixFixe. The top-performing solutions in the challenge exceeded performance of all previous state-of-the-art models for similar data. </a> </dd> </p>
<p>  <dt><a href="https://github.com/ikwak2/DCLEAR">R/DCLEAR</a></dt>
  <dd> R/DCLEAR is an R package for Distance based Cell LinEAge Reconstruction(DCLEAR). These codes are created during the participation of <a href="https://www.synapse.org/#!Synapse:syn20692755/wiki/595096">Cell Lineage Reconstruction DREAM challenge.</a> </dd> </p>
<p>
  <dt><a href="https://cran.r-project.org/web/packages/DrImpute/index.html">R/DrImpute</a></dt>
  <dd> R codes for imputing dropout events. Many statistical methods in cell type identification, visualization and lineage reconstruction do not account for dropout events ('PCAreduce', 'SC3', 'PCA', 't-SNE', 'Monocle', 'TSCAN', etc). 'DrImpute' can improve the performance of such software by imputing dropout events. </dd> </p>
<p>  <dt><a href="https://cran.r-project.org/web/packages/aSPU/index.html">R/aSPU</a></dt>
  <dd> R codes for the (adaptive) Sum of Powered Score ('SPU' and 'aSPU')
    tests, inverse variance weighted Sum of Powered score ('SPUw' and 'aSPUw') tests
    and gene-based and some pathway based association tests (Pathway based Sum of
    Powered Score tests ('SPUpath'), adaptive 'SPUpath' ('aSPUpath') test, 'GEEaSPU'
    test for multiple traits - single 'SNP' (single-nucleotide-polymorphism)
    association in generalized estimation equations, 'MTaSPUs' test for multiple
    traits - single 'SNP' association with Genome Wide Association Studies ('GWAS')
    summary statistics, Gene-based Association Test that uses an extended Simes
    procedure ('GATES'), Hybrid Set-based Test ('HYST') and extended version
    of 'GATES' test for pathway-based association testing ('GATES-Simes'). ).
    The tests can be used with genetic and other data sets with covariates. The
    response variable is binary or quantitative. Summary; (1) Single trait-'SNP' set
    association with individual-level data ('aSPU', 'aSPUw'), (2) Single trait-'SNP'
    set association with summary statistics ('aSPUs'), (3) Single trait-pathway
    association with individual-level data ('aSPUpath'), (4) Single trait-pathway
    association with summary statistics ('aSPUsPath'), (5) Multiple traits-single
    'SNP' association with individual-level data ('GEEaSPU'), (6) Multiple traits-
    single SNP association with summary statistics ('MTaSPUs'), (7) Multiple traits-`SNP' set association with summary statistics(`MTaSPUsSet'), (8) Multiple traits-pathway association with summary statistics(`MTaSPUsSetPath'). </dd> </p>
<p>  <dt><a href="https://github.com/ikwak2/funqtl">R/funqtl</a></dt>
  <dd> An R package
with add-on functions for
the <a href="https://github.com/kbroman/qtl">R/qtl</a> package to deal with function
valued trait on QTL mapping. </dd> </p>
<p>  <dt><a href="https://github.com/ikwak2/xoi">R/xoi</a> </dt>
  <dd>Tools for analyzing crossover interference. Assisted Karl Broman
  to develop the package. </dd> </p>
<p>  <dt><a href="https://github.com/ikwak2/APCP">R/APCP</a></dt>
  <dd> An R pacakage that plots Andrews type Parallel Coordinate Plot </dd></p>
</dl>

</tr>
</table>

<p class=noindent>
<hr size = "7" noshade="noshade" />
<p class=noindent>
<table cellspacing="0" cellpadding="0" width=100%>
  <tr>
    <td align="left"> <p class=noindent>
      <a href="../index.html">Back to home page</a> </p> </td>
    <td align="right"> <img src="../CAU_ET_logo.png" height=100 alt="Lablogo"
			title="Lablogo"> </td>
</tr>
</table>

  </body>
</html>
