<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />
<meta name="viewport" content="width=device-width, initial-scale=1">

<meta name="author" content="Il-Youp Kwak" />


<title>Performing MTaSPUsSet (aSPUs) for multiple genes</title>



<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>


<link href="data:text/css;charset=utf-8,body%20%7B%0Abackground%2Dcolor%3A%20%23fff%3B%0Amargin%3A%201em%20auto%3B%0Amax%2Dwidth%3A%20700px%3B%0Aoverflow%3A%20visible%3B%0Apadding%2Dleft%3A%202em%3B%0Apadding%2Dright%3A%202em%3B%0Afont%2Dfamily%3A%20%22Open%20Sans%22%2C%20%22Helvetica%20Neue%22%2C%20Helvetica%2C%20Arial%2C%20sans%2Dserif%3B%0Afont%2Dsize%3A%2014px%3B%0Aline%2Dheight%3A%201%2E35%3B%0A%7D%0A%23header%20%7B%0Atext%2Dalign%3A%20center%3B%0A%7D%0A%23TOC%20%7B%0Aclear%3A%20both%3B%0Amargin%3A%200%200%2010px%2010px%3B%0Apadding%3A%204px%3B%0Awidth%3A%20400px%3B%0Aborder%3A%201px%20solid%20%23CCCCCC%3B%0Aborder%2Dradius%3A%205px%3B%0Abackground%2Dcolor%3A%20%23f6f6f6%3B%0Afont%2Dsize%3A%2013px%3B%0Aline%2Dheight%3A%201%2E3%3B%0A%7D%0A%23TOC%20%2Etoctitle%20%7B%0Afont%2Dweight%3A%20bold%3B%0Afont%2Dsize%3A%2015px%3B%0Amargin%2Dleft%3A%205px%3B%0A%7D%0A%23TOC%20ul%20%7B%0Apadding%2Dleft%3A%2040px%3B%0Amargin%2Dleft%3A%20%2D1%2E5em%3B%0Amargin%2Dtop%3A%205px%3B%0Amargin%2Dbottom%3A%205px%3B%0A%7D%0A%23TOC%20ul%20ul%20%7B%0Amargin%2Dleft%3A%20%2D2em%3B%0A%7D%0A%23TOC%20li%20%7B%0Aline%2Dheight%3A%2016px%3B%0A%7D%0Atable%20%7B%0Amargin%3A%201em%20auto%3B%0Aborder%2Dwidth%3A%201px%3B%0Aborder%2Dcolor%3A%20%23DDDDDD%3B%0Aborder%2Dstyle%3A%20outset%3B%0Aborder%2Dcollapse%3A%20collapse%3B%0A%7D%0Atable%20th%20%7B%0Aborder%2Dwidth%3A%202px%3B%0Apadding%3A%205px%3B%0Aborder%2Dstyle%3A%20inset%3B%0A%7D%0Atable%20td%20%7B%0Aborder%2Dwidth%3A%201px%3B%0Aborder%2Dstyle%3A%20inset%3B%0Aline%2Dheight%3A%2018px%3B%0Apadding%3A%205px%205px%3B%0A%7D%0Atable%2C%20table%20th%2C%20table%20td%20%7B%0Aborder%2Dleft%2Dstyle%3A%20none%3B%0Aborder%2Dright%2Dstyle%3A%20none%3B%0A%7D%0Atable%20thead%2C%20table%20tr%2Eeven%20%7B%0Abackground%2Dcolor%3A%20%23f7f7f7%3B%0A%7D%0Ap%20%7B%0Amargin%3A%200%2E5em%200%3B%0A%7D%0Ablockquote%20%7B%0Abackground%2Dcolor%3A%20%23f6f6f6%3B%0Apadding%3A%200%2E25em%200%2E75em%3B%0A%7D%0Ahr%20%7B%0Aborder%2Dstyle%3A%20solid%3B%0Aborder%3A%20none%3B%0Aborder%2Dtop%3A%201px%20solid%20%23777%3B%0Amargin%3A%2028px%200%3B%0A%7D%0Adl%20%7B%0Amargin%2Dleft%3A%200%3B%0A%7D%0Adl%20dd%20%7B%0Amargin%2Dbottom%3A%2013px%3B%0Amargin%2Dleft%3A%2013px%3B%0A%7D%0Adl%20dt%20%7B%0Afont%2Dweight%3A%20bold%3B%0A%7D%0Aul%20%7B%0Amargin%2Dtop%3A%200%3B%0A%7D%0Aul%20li%20%7B%0Alist%2Dstyle%3A%20circle%20outside%3B%0A%7D%0Aul%20ul%20%7B%0Amargin%2Dbottom%3A%200%3B%0A%7D%0Apre%2C%20code%20%7B%0Abackground%2Dcolor%3A%20%23f7f7f7%3B%0Aborder%2Dradius%3A%203px%3B%0Acolor%3A%20%23333%3B%0Awhite%2Dspace%3A%20pre%2Dwrap%3B%20%0A%7D%0Apre%20%7B%0Aborder%2Dradius%3A%203px%3B%0Amargin%3A%205px%200px%2010px%200px%3B%0Apadding%3A%2010px%3B%0A%7D%0Apre%3Anot%28%5Bclass%5D%29%20%7B%0Abackground%2Dcolor%3A%20%23f7f7f7%3B%0A%7D%0Acode%20%7B%0Afont%2Dfamily%3A%20Consolas%2C%20Monaco%2C%20%27Courier%20New%27%2C%20monospace%3B%0Afont%2Dsize%3A%2085%25%3B%0A%7D%0Ap%20%3E%20code%2C%20li%20%3E%20code%20%7B%0Apadding%3A%202px%200px%3B%0A%7D%0Adiv%2Efigure%20%7B%0Atext%2Dalign%3A%20center%3B%0A%7D%0Aimg%20%7B%0Abackground%2Dcolor%3A%20%23FFFFFF%3B%0Apadding%3A%202px%3B%0Aborder%3A%201px%20solid%20%23DDDDDD%3B%0Aborder%2Dradius%3A%203px%3B%0Aborder%3A%201px%20solid%20%23CCCCCC%3B%0Amargin%3A%200%205px%3B%0A%7D%0Ah1%20%7B%0Amargin%2Dtop%3A%200%3B%0Afont%2Dsize%3A%2035px%3B%0Aline%2Dheight%3A%2040px%3B%0A%7D%0Ah2%20%7B%0Aborder%2Dbottom%3A%204px%20solid%20%23f7f7f7%3B%0Apadding%2Dtop%3A%2010px%3B%0Apadding%2Dbottom%3A%202px%3B%0Afont%2Dsize%3A%20145%25%3B%0A%7D%0Ah3%20%7B%0Aborder%2Dbottom%3A%202px%20solid%20%23f7f7f7%3B%0Apadding%2Dtop%3A%2010px%3B%0Afont%2Dsize%3A%20120%25%3B%0A%7D%0Ah4%20%7B%0Aborder%2Dbottom%3A%201px%20solid%20%23f7f7f7%3B%0Amargin%2Dleft%3A%208px%3B%0Afont%2Dsize%3A%20105%25%3B%0A%7D%0Ah5%2C%20h6%20%7B%0Aborder%2Dbottom%3A%201px%20solid%20%23ccc%3B%0Afont%2Dsize%3A%20105%25%3B%0A%7D%0Aa%20%7B%0Acolor%3A%20%230033dd%3B%0Atext%2Ddecoration%3A%20none%3B%0A%7D%0Aa%3Ahover%20%7B%0Acolor%3A%20%236666ff%3B%20%7D%0Aa%3Avisited%20%7B%0Acolor%3A%20%23800080%3B%20%7D%0Aa%3Avisited%3Ahover%20%7B%0Acolor%3A%20%23BB00BB%3B%20%7D%0Aa%5Bhref%5E%3D%22http%3A%22%5D%20%7B%0Atext%2Ddecoration%3A%20underline%3B%20%7D%0Aa%5Bhref%5E%3D%22https%3A%22%5D%20%7B%0Atext%2Ddecoration%3A%20underline%3B%20%7D%0A%0Acode%20%3E%20span%2Ekw%20%7B%20color%3A%20%23555%3B%20font%2Dweight%3A%20bold%3B%20%7D%20%0Acode%20%3E%20span%2Edt%20%7B%20color%3A%20%23902000%3B%20%7D%20%0Acode%20%3E%20span%2Edv%20%7B%20color%3A%20%2340a070%3B%20%7D%20%0Acode%20%3E%20span%2Ebn%20%7B%20color%3A%20%23d14%3B%20%7D%20%0Acode%20%3E%20span%2Efl%20%7B%20color%3A%20%23d14%3B%20%7D%20%0Acode%20%3E%20span%2Ech%20%7B%20color%3A%20%23d14%3B%20%7D%20%0Acode%20%3E%20span%2Est%20%7B%20color%3A%20%23d14%3B%20%7D%20%0Acode%20%3E%20span%2Eco%20%7B%20color%3A%20%23888888%3B%20font%2Dstyle%3A%20italic%3B%20%7D%20%0Acode%20%3E%20span%2Eot%20%7B%20color%3A%20%23007020%3B%20%7D%20%0Acode%20%3E%20span%2Eal%20%7B%20color%3A%20%23ff0000%3B%20font%2Dweight%3A%20bold%3B%20%7D%20%0Acode%20%3E%20span%2Efu%20%7B%20color%3A%20%23900%3B%20font%2Dweight%3A%20bold%3B%20%7D%20%20code%20%3E%20span%2Eer%20%7B%20color%3A%20%23a61717%3B%20background%2Dcolor%3A%20%23e3d2d2%3B%20%7D%20%0A" rel="stylesheet" type="text/css" />

</head>

<body>



<div class="fluid-row" id="header">


<h1 class="title">Performing MTaSPUsSet (aSPUs) for multiple genes</h1>
<h4 class="author"><em>Il-Youp Kwak</em></h4>
<h4 class="date"><em>2016-5-3</em></h4>

</div>


<p>We mapped 34 rs-ids from <a href="mappingSnpToGene2.html">Mapping Snp to Gene</a> tutorial. <a href="myrsidout.genes.annot" class="uri">myrsidout.genes.annot</a></p>
<p>I prepared 6 multi-trait summary statistics data for 34 rs-ids, <a href="data/exdat.txt">exdat.txt</a>.</p>
<pre><code>bash-3.2$ head exdat.txt
rs3121561       1       990380  0.38    0.14    0.012   0.070   0.64    0.81
rs2799064       1       957898  0.58    0.18    0.61    0.79    0.83    0.73
rs3128126       1       962210  0.076   0.12    0.081   0.17    0.15    0.81
rs3766186       1       1162435 0.29    0.57    0.32    0.73    0.52    0.63
rs11721 1       1152631 0.15    0.34    0.13    0.68    0.17    0.25
rs2887286       1       1156131 0.29    0.81    0.19    0.30    0.22    0.78
rs3813200       1       1151300 0.14    0.41    0.46    0.43    0.26    0.77
rs7515488       1       1163804 0.13    0.87    0.11    0.96    0.42    0.11
rs3813199       1       1158277 0.20    0.64    0.27    0.65    0.40    0.65
rs6603781       1       1158631 0.33    0.68    0.90    0.37    0.35    0.79
</code></pre>
<p>I made a Perl program, <a href="perl/processGene.pl" class="uri">processGene.pl</a>, to process magma outcome, <a href="data/myrsidout.genes.annot">myrsidout.genes.annot</a>. The Usage is : ./processGene.pl “Magma Result” “Folder Name”</p>
<pre><code>bash-3.2$ ./processGene.pl myrsidout.genes.annot exGenes
SSU72  25</code></pre>
<p>It also shows a Gene name with maximum number of SNPs are mapped. The biggest gene in our example is <em>SSU72</em> which have 25 SNPs mapped. The program make <code>exGenes</code> directory and save results there.</p>
<pre><code>bash-3.2$ cd exGenes
bash-3.2$ ls
AGRN.txt  SDF4.txt  SSU72.txt  TMEM240.txt
bash-3.2$ head AGRN.txt
rs3121561
rs2799064
rs3128126</code></pre>
<p>As you can check from the code above. <a href="data/AGRN.txt">AGRN.txt</a> file have rs-ids mapped.</p>
<p>After running this perl code, I write a R function as follows. Take gene <em>AGRN</em> for example to explain. You can change gene name in the code for other genes.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">library</span>(aSPU)
g &lt;-<span class="st"> &quot;AGRN&quot;</span></code></pre></div>
<p>Assume we already calculated correlation between phenotypes from the whole population. <a href="data/exvarPhe.txt">exvarPhe.txt</a></p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">varPhe &lt;-<span class="st"> </span><span class="kw">as.matrix</span>(<span class="kw">read.table</span>(<span class="st">&quot;exvarPhe.txt&quot;</span>))
varPhe</code></pre></div>
<pre><code>##                V1    V2    V3    V4   V5    V6
##        [1,]  1.00 -0.01  0.19  0.30 0.66  0.03
##        [2,] -0.01  1.00 -0.01 -0.01 0.04 -0.01
##        [3,]  0.19 -0.01  1.00  0.29 0.21 -0.02
##        [4,]  0.30 -0.01  0.29  1.00 0.29  0.32
##        [5,]  0.66  0.04  0.21  0.29 1.00  0.02
##        [6,]  0.03 -0.01 -0.02  0.32 0.02  1.00</code></pre>
<p>Using awk command I took subdata for <em>AGRN</em> on <a href="data/subdatAGRN.txt">subdatAGRN.txt</a> from <a href="data/exdat.txt">exdat.txt</a>.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">    tof &lt;-<span class="st"> </span><span class="kw">paste</span>(<span class="st">&quot;subdat&quot;</span>,g,<span class="st">&quot;.txt&quot;</span>,<span class="dt">sep =</span><span class="st">&quot;&quot;</span>)
    cmd2run &lt;-<span class="st"> </span><span class="kw">paste</span>(<span class="st">&quot;awk 'NR==FNR {h[$1] = $1; next} h[$1] !=</span><span class="ch">\&quot;\&quot;</span><span class="st"> {print $0}' exGenes/&quot;</span>,g,<span class="st">&quot;.txt exdat.txt &gt;&quot;</span>, tof, <span class="dt">sep=</span><span class="st">&quot;&quot;</span>)
    <span class="kw">system</span>(cmd2run)

    sdat &lt;-<span class="st"> </span><span class="kw">read.table</span>(<span class="dt">file=</span>tof, <span class="dt">colClasses=</span><span class="kw">c</span>(<span class="st">&quot;character&quot;</span>, <span class="st">&quot;numeric&quot;</span>, <span class="st">&quot;numeric&quot;</span>, <span class="st">&quot;numeric&quot;</span>, <span class="st">&quot;numeric&quot;</span>, <span class="st">&quot;numeric&quot;</span>, <span class="st">&quot;numeric&quot;</span>, <span class="st">&quot;numeric&quot;</span>, <span class="st">&quot;numeric&quot;</span>, <span class="st">&quot;character&quot;</span>, <span class="st">&quot;character&quot;</span>)  )
        <span class="kw">system</span>(<span class="kw">paste</span>(<span class="st">&quot;rm&quot;</span>,tof) )</code></pre></div>
<p>Again, using awk command, I took subdata for <em>AGRN</em>, <a href="data/tempdataAGRN.txt">tempdataAGRN.txt</a>, from reference panel, <a href="data/SubsetCh1.txt">SubsetCh1.txt</a>.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">    snpids &lt;-<span class="st"> </span>sdat[,<span class="dv">1</span>]
    tempf =<span class="st"> </span><span class="kw">paste</span>(<span class="st">&quot;tempids&quot;</span>,g,<span class="st">&quot;.txt&quot;</span> , <span class="dt">sep=</span><span class="st">&quot;&quot;</span>)
    <span class="kw">write.table</span>(snpids,  <span class="dt">file =</span> tempf, <span class="dt">row.names=</span><span class="ot">FALSE</span>, <span class="dt">col.names=</span><span class="ot">FALSE</span>, <span class="dt">quote=</span><span class="ot">FALSE</span> )
    tof &lt;-<span class="st"> </span><span class="kw">paste</span>(<span class="st">&quot;tempdata&quot;</span>,g,<span class="st">&quot;.txt&quot;</span>, <span class="dt">sep=</span><span class="st">&quot;&quot;</span>)

    <span class="kw">system</span>( <span class="kw">paste</span>(<span class="st">&quot;awk 'NR==FNR {h[$1] = $0; next} h[$1] !=</span><span class="ch">\&quot;\&quot;</span><span class="st"> {print h[$1]}' SubsetCh1.txt &quot;</span>, tempf,<span class="st">&quot; &gt; &quot;</span>, tof, <span class="dt">sep=</span><span class="st">&quot;&quot;</span>) )

    locdat &lt;-<span class="st"> </span><span class="kw">read.table</span>(<span class="dt">file=</span>tof)
    datn &lt;-<span class="st"> </span><span class="kw">t</span>(<span class="kw">matrix</span>(<span class="kw">as.numeric</span>(<span class="kw">as.matrix</span>(locdat[,<span class="dv">3</span>:<span class="dv">381</span>])), <span class="kw">dim</span>(locdat)[<span class="dv">1</span>],<span class="dv">379</span>))

    <span class="kw">colnames</span>(datn) &lt;-<span class="st"> </span>locdat[,<span class="dv">1</span>]</code></pre></div>
<p>(Note: The reason I used awk is because of the speed. It is small example so reading data in R and processing it in R is fast. However, the 1000 human genome project data have 81.2 million polymorphic markers. It takes really long time to read data in R. I tried it once and terminated R. On the other hand, awk do the same job in a few seconds. )</p>
<p>Save used allele in reference panel.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">    usedallele &lt;-<span class="st"> </span><span class="kw">as.character</span>(locdat[,<span class="dv">2</span>])
    <span class="kw">system</span>(<span class="kw">paste</span>(<span class="st">&quot;rm&quot;</span>,tof))</code></pre></div>
<p>If any column have all 0’s, 1’s or 2’s, erase it.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">    snpids1 &lt;-<span class="st"> </span>snpids[(snpids %in%<span class="st"> </span><span class="kw">colnames</span>(datn))]

    a =<span class="st"> </span><span class="kw">max</span>(<span class="kw">apply</span>( datn ==<span class="st"> </span><span class="dv">0</span>, <span class="dv">2</span>, sum )) ==<span class="st"> </span><span class="dv">379</span>
    b =<span class="st"> </span><span class="kw">max</span>(<span class="kw">apply</span>( datn ==<span class="st"> </span><span class="dv">1</span>, <span class="dv">2</span>, sum )) ==<span class="st"> </span><span class="dv">379</span>
    c =<span class="st"> </span><span class="kw">max</span>(<span class="kw">apply</span>( datn ==<span class="st"> </span><span class="dv">2</span>, <span class="dv">2</span>, sum )) ==<span class="st"> </span><span class="dv">379</span>

    if( a |<span class="st"> </span>b |<span class="st"> </span>c ) {
        eraseit =<span class="st"> </span><span class="dv">0</span>
        eraseit &lt;-<span class="st"> </span><span class="kw">c</span>(eraseit, <span class="kw">which</span>( <span class="kw">apply</span>( datn ==<span class="st"> </span><span class="dv">0</span>, <span class="dv">2</span>, sum ) ==<span class="st"> </span><span class="dv">112</span> ),
                                <span class="kw">which</span>(<span class="kw">apply</span>( datn ==<span class="st"> </span><span class="dv">1</span>, <span class="dv">2</span>, sum ) ==<span class="st"> </span><span class="dv">112</span>),
                                <span class="kw">which</span>(<span class="kw">apply</span>( datn ==<span class="st"> </span><span class="dv">2</span>, <span class="dv">2</span>, sum ) ==<span class="st"> </span><span class="dv">112</span>) )

        datn &lt;-<span class="st"> </span>datn[, -eraseit]
        snpids1 &lt;-<span class="st"> </span>snpids1[-eraseit]
        usedallele &lt;-<span class="st"> </span>usedallele[-eraseit]
    }</code></pre></div>
<p>Compare allele used for 0,1,2 coding for original data and reference panel data. Alter 0 and 2 if different alleles are used.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">    <span class="kw">rownames</span>(sdat) &lt;-<span class="st"> </span>sdat[,<span class="dv">1</span>]
    allele &lt;-<span class="st"> </span><span class="kw">toupper</span>(sdat[snpids1, <span class="dv">11</span>])
    difal &lt;-<span class="st"> </span><span class="kw">ifelse</span>( allele ==<span class="st"> </span>usedallele, <span class="dv">1</span>, -<span class="dv">1</span>)
    nsdat &lt;-<span class="st"> </span>sdat[snpids1,]

    subdat2 &lt;-<span class="st"> </span><span class="kw">as.matrix</span>(datn)
    temp &lt;-<span class="st"> </span>subdat2[, difal ==<span class="st"> </span>-<span class="dv">1</span> ]
    temp[ temp ==<span class="st"> </span><span class="dv">0</span>] =<span class="st"> </span><span class="dv">3</span>
    temp[ temp ==<span class="st"> </span><span class="dv">2</span>] =<span class="st"> </span><span class="dv">0</span>
    temp[ temp ==<span class="st"> </span><span class="dv">3</span>] =<span class="st"> </span><span class="dv">2</span>
    subdat2[, difal ==<span class="st"> </span>-<span class="dv">1</span> ]  &lt;-<span class="st"> </span>temp</code></pre></div>
<p>Calculate correlation matrix</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">    datn &lt;-<span class="st"> </span><span class="kw">as.matrix</span>(subdat2)
    datn &lt;-<span class="st"> </span><span class="kw">as.matrix</span>(datn)
    corSNP &lt;-<span class="st"> </span><span class="kw">cor</span>(<span class="kw">as.matrix</span>(datn), <span class="dt">use=</span><span class="st">&quot;pairwise.complete.obs&quot;</span>)

    Ps &lt;-<span class="st"> </span><span class="kw">as.matrix</span>(nsdat[,<span class="kw">c</span>(<span class="dv">4</span>:<span class="dv">9</span>)])</code></pre></div>
<p>Erase rs-ids with high correlation, <span class="math inline">\(&gt;\)</span> 0.95 and <span class="math inline">\(&lt;\)</span> -0.95.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">    while( <span class="kw">sum</span>( corSNP &lt;<span class="st"> </span>-<span class="fl">0.95</span> ) &gt;<span class="st"> </span><span class="dv">0</span> ) {
       toerase &lt;-<span class="st"> </span><span class="kw">which</span>( <span class="kw">apply</span>( corSNP &lt;<span class="st"> </span>-.<span class="dv">95</span>, <span class="dv">1</span>, sum) ==<span class="st"> </span><span class="kw">max</span>( <span class="kw">apply</span>( corSNP &lt;<span class="st"> </span>-.<span class="dv">95</span>, <span class="dv">1</span>, sum) ) )
       corSNP &lt;-<span class="st"> </span>corSNP[-toerase[<span class="dv">1</span>], -toerase[<span class="dv">1</span>]]
       Ps &lt;-<span class="st"> </span>Ps[-toerase[<span class="dv">1</span>],]
       if( <span class="kw">is.matrix</span>(corSNP) ==<span class="st"> </span><span class="ot">FALSE</span>) break
    }

    corSNP =<span class="st"> </span><span class="kw">as.matrix</span>(corSNP)
    while( <span class="kw">sum</span>( corSNP &gt;<span class="st"> </span><span class="fl">0.95</span> ) &gt;<span class="st"> </span><span class="kw">ncol</span>(corSNP) ) {
       toerase &lt;-<span class="st"> </span><span class="kw">which</span>( <span class="kw">apply</span>( corSNP &gt;<span class="st"> </span><span class="fl">0.95</span>, <span class="dv">1</span>, sum) ==<span class="st"> </span><span class="kw">max</span>( <span class="kw">apply</span>( corSNP &gt;<span class="st"> </span><span class="fl">0.95</span>, <span class="dv">1</span>, sum) ) )
       corSNP &lt;-<span class="st"> </span>corSNP[-toerase[<span class="dv">1</span>], -toerase[<span class="dv">1</span>]]
       Ps &lt;-<span class="st"> </span>Ps[-toerase[<span class="dv">1</span>],]
       if( <span class="kw">is.matrix</span>(corSNP) ==<span class="st"> </span><span class="ot">FALSE</span>) break
    }

    Ps =<span class="st"> </span><span class="kw">as.matrix</span>(Ps)</code></pre></div>
<p>Perform MTaSPUsSet test.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="co">#    out &lt;- MTaSPUsSet(Ps, corSNP=corSNP, corPhe = varPhe, pow=c(1,2,4,8),  pow2 = c(1,2,4,8), n.perm=10^3, Ps=TRUE)</span>
    out &lt;-<span class="st"> </span><span class="kw">MTaSPUsSetC</span>(Ps, <span class="dt">corSNP=</span>corSNP, <span class="dt">corPhe =</span> varPhe, <span class="dt">pow=</span><span class="kw">c</span>(<span class="dv">1</span>,<span class="dv">2</span>,<span class="dv">4</span>,<span class="dv">8</span>),  <span class="dt">pow2 =</span> <span class="kw">c</span>(<span class="dv">1</span>,<span class="dv">2</span>,<span class="dv">4</span>,<span class="dv">8</span>), <span class="dt">n.perm=</span><span class="dv">10</span>^<span class="dv">3</span>, <span class="dt">Ps=</span><span class="ot">TRUE</span>)
out</code></pre></div>
<pre><code>## MTSPUs1,1 MTSPUs2,1 MTSPUs4,1 MTSPUs8,1 MTSPUs1,2 MTSPUs2,2 MTSPUs4,2 MTSPUs8,2
##     0.145     0.174     0.247     0.313     0.140     0.201     0.253     0.272
## MTSPUs1,4 MTSPUs2,4 MTSPUs4,4 MTSPUs8,4 MTSPUs1,8 MTSPUs2,8 MTSPUs4,8 MTSPUs8,8
##     0.161     0.203     0.202     0.193     0.177     0.168     0.165     0.164
##   MTaSPUs
##     0.236</code></pre>
<p>Perform aSPUs test.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">Out &lt;-<span class="ot">NULL</span>
for(i in <span class="dv">1</span>:<span class="dv">6</span>) {
   Out[[i]] &lt;-<span class="st"> </span><span class="kw">aSPUs</span>(Ps[,i], <span class="dt">corrSNP =</span> corSNP, <span class="dt">pow=</span><span class="kw">c</span>(<span class="dv">1</span>,<span class="dv">2</span>,<span class="dv">4</span>,<span class="dv">8</span>), <span class="dt">n.perm =</span> <span class="dv">100</span>, <span class="dt">Ps=</span><span class="ot">TRUE</span>)$pvs
}
Out</code></pre></div>
<pre><code>## [[1]]
##     SPUs1     SPUs2     SPUs4     SPUs8     aSPUs
## 0.2400000 0.2200000 0.2100000 0.1800000 0.2376238
## [[2]]
##      SPUs1      SPUs2      SPUs4      SPUs8      aSPUs
## 0.05000000 0.09000000 0.11000000 0.19000000 0.06930693
## [[3]]
##      SPUs1      SPUs2      SPUs4      SPUs8      aSPUs
## 0.07000000 0.03000000 0.01000000 0.01000000 0.02970297
## [[4]]
##     SPUs1     SPUs2     SPUs4     SPUs8     aSPUs
## 0.1700000 0.1400000 0.1500000 0.1500000 0.1683168
## [[5]]
##     SPUs1     SPUs2     SPUs4     SPUs8     aSPUs
## 0.5900000 0.5000000 0.4300000 0.3900000 0.4554455
## [[6]]
##     SPUs1     SPUs2     SPUs4     SPUs8     aSPUs
## 0.9200000 0.9300000 0.9600000 0.9700000 0.9306931</code></pre>
<p>You can change <code>g</code> to calculate MTaSPUsSet/aSPUs P-values for other genes.</p>



<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
