<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
          "http://www.w3.org/TR/html1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

  <head>
    <title> Tutorials </title>
    <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
    <link rel="stylesheet" type="text/css" href="../iystyle2.css" />

  </head>

    <h1> Tutorials </h1>
<hr size = "7" noshade="noshade" />



<table noborder width="100%" cellspacing="50" cellpadding="0">
  <tr> <td valign="top" align="left">

      
<dl>

    <h4 >On R/aSPU</h4>

<p>  <dt><a href="mappingSnpToGene2.html">Mapping Snp to Gene</a></dt>
  <dd> Assume you have rs-ids, and you want to map rs-ids to gene for gene based analysis. This page illustrate one way to do it. </dd> </p>
<p>  <dt><a href="CorrFromRef.html">Getting correlation estimate among SNPs from reference panel</a></dt>
  <dd> This page illustrate how one can get correlation estimate among SNPs from the reference panel. </dd> </p>
<p>  <dt><a href="ForMTgenes.html">R and Perl codes to perform aSPUs and MTaSPUsSet</a> </dt>
  <dd> Assume we have multi-trait summary statistics data for mapped SNPs from <a href="mappingSnpToGene2.html">Mapping Snp to Gene</a>, this page illustrate how to perform aSPUs or MTaSPUsSet test using Summary Statistics data</dd> </p>
<p>  <dt><a href="SpeedComp.html">Speed comparison using R, awk and Perl</a> </dt>
  <dd> I used `awk` in some tutorials. I did some speed comparisons.</dd></p>
<p>  <dt><a href="pathwayMani.html">Pathway data manipulation</a> </dt>
  <dd> Where to get pathways(or gene sets) and how to manipulate.</dd></p>
<p>  <dt><a href="aSPUstat.html">Vignette for aSPUs and aSPUsPath</a></dt>
  <dd> Adaptive Gene- and Pathway- Trait Association testing with GWAS Summary Statistics (aSPUs() and aSPUsPath()) </dd></p>
    <p>  <dt><a href="MTaSPUsSet.html">Vignette for MTaSPUsSet</a> </dt>
  <dd> Adaptive Gene - Multitrait Association testing with GWAS Summary Statistics (MTaSPUsSet() ) </dd></p>
</dl>

    <h4 >On R/funqtl</h4>

<p>  <dt><a href="funqtl.html">Vignette for funqtl</a></dt>
  <dd> This vignette illustrates the use of the funqtl package for QTL mapping with function-valued traits (e.g., growth measured over time). </dd> </p>
<p>  <dt><a href="fun2mult.html">Vignette for fun2mult</a></dt>
  <dd> This vignette illustrates the use of the funqtl package for QTL mapping with function-valued traits using functional principal component analysis and multi-trait mapping. </dd> </p>


  <hr size = "3" noshade="noshade" />
  
Part of the data here downloaded from <a href="https://www.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium_data_files">GIANT consortium data files</a> website. Data is from <a href="http://www.ncbi.nlm.nih.gov/pubmed/23754948?dopt=Citation"> Randall JC, Winkler TW, Kutalik Z, Berndt SI, Jackson AU, Monda KL, Kilpeläinen TO, Esko T, Mägi R, Li S, et al.</a> (2013). Sex-stratified genome-wide association studies including 270,000 individuals show sexual dimorphism in genetic loci for anthropometric traits. PLoS Genet 9: e1003500.

  
</tr>
</table>

 <p class=noindent>
<hr size = "7" noshade="noshade" />
<p class=noindent>
<table cellspacing="0" cellpadding="0" width=100%>
  <tr>
    <td align="left"> <p class=noindent>
      <a href="http://creativecommons.org/publicdomain/zero/1.0/"><img src="http://i.creativecommons.org/p/zero/1.0/88x31.png" alt="CC0"/></a> &nbsp;
      <a href="../index.html">Back to home page</a> </p> </td>
    <td align="right"> <img src="../CAU_ET_logo.png" height=100 alt="Lablogo"
			title="Lablogo"> </td>
</tr>
</table>

    

  </body>
</html>
