<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
          "http://www.w3.org/TR/html1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

  <head>
    <title> Projects and Publications </title>
    <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
    <link rel="stylesheet" type="text/css" href="../iystyle2.css" />

  </head>

    <h1> Projects </h1> 
<hr size = "7" noshade="noshade" />

<table noborder width="100%" cellspacing="50" cellpadding="0">
  <tr> <td valign="top" align="left">

<p class=large> 

  <dt> 한국연구재단 신진연구 신규과제 (2020.3 ~ 2023.2) </dt>
    <dd> Deep Learning Applications in Voice Security </dd> 
        
</tr>
</table>

<h1> Publications </h1>

<hr size = "7" noshade="noshade" />

<table noborder width="100%" cellspacing="50" cellpadding="0">
<tr> <td valign="top" align="left">

   <h4 id="2022">2022</h4>
<p> Gong W, Kim H, Garry D, <font color="#FF7975">Kwak IY</font> (2022) Single cell lineage reconstruction using distance-based algorithms and the R package, DCLEAR <a href="https://bmcbioinformatics.biomedcentral.com/">BMC Bioinformatics</a>, 23(1), 1-14 <br>
    [ <a href="https://pubmed.ncbi.nlm.nih.gov/35331133/">PubMed</a> |<a href="https://github.com/ikwak2/DCLEAR">R/DCLEAR software</a> | <a href="https://link.springer.com/content/pdf/10.1186/s12859-022-04633-x.pdf">pdf</a> ] </p>
<p> Yang J, Choi S, Lee Y, Oh S, <font color="#FF7975">Kwak IY</font> (2022) Light-Weight Frequency Information Aware Neural Network Architecture for Voice Spoofing Detection, <a href="https://www.icpr2022.com/"> 26th International Conference on
Pattern Recognition (ICPR 2022)</a>, Montréal Québec, Canada.
<br>
  [ <a href="https://ieeexplore.ieee.org/document/9956079">IEEE Xplore</a> | <a href="yang_icpr.pdf">pdf</a> ] </p>
<p> Choi S, <font color="#FF7975">Kwak IY</font>, Oh S (2022) Overlapped Frequency-Distributed Network: Frequency-Aware Voice Spoofing Countermeasure, <a href="https://interspeech2022.org/"> Interspeech 2022</a>, Incheon, Korea.
<br>
  [ <a href="https://www.isca-speech.org/archive/interspeech_2022/choi22c_interspeech.html">Link</a> | <a href="https://www.isca-speech.org/archive/pdfs/interspeech_2022/choi22c_interspeech.pdf">pdf</a> ] </p>
<p> <font color="#FF7975">Kwak IY</font>, Choi S, Yang J, Lee Y, Oh S (2022) Low-quality Fake Audio Detection through Frequency Feature Masking, <a href="https://2022.acmmm.org/"> 30th ACM International Conference on Multimedia (ACM MM 2022, Workshop on Deepfake Detection for Audio Multimedia)</a>, Lisbon, Portugal.
<br>
  [ <a href="https://dl.acm.org/doi/abs/10.1145/3552466.3556533">ACM DL</a> | <a href="ddam004-kwak.pdf">pdf</a> ] </p>
<p> Lee J, Kang T, Kim N, Han S, Won H, Gong W, <font color="#FF7975">Kwak IY</font> (2022) Deep Learning Based Heart Murmur Detection using Frequency-time Domain Features of Heartbeat Sounds, <a href="https://events.tuni.fi/cinc2022/"> The 49th Computing in Cardiology conference (CinC 2022)</a>, Tampere, Finland.
<br>
  [ <a href=https://cinc.org/2022/Program/accepted/71_Preprint.pdf>preprint</a> ] </p>
<p> Yoshihara T, Miller ND, Rabanal FA, Myles H, <font color="#FF7975">Kwak IY</font>, Broman KW, Boris S, Ivan B, Brian PD, Matthew EH, Edgar PS (2022) Leveraging orthology within maize and Arabidopsis QTL to identify genes affecting natural variation in gravitropism, <a href="https://www.pnas.org/"> Proceedings of the National Academy of Sciences of the United States of America</a>, 119 (40) e2212199119.
<br>
  [ <a href="https://pubmed.ncbi.nlm.nih.gov/36161933/">PubMed</a> | <a href="https://www.pnas.org/doi/epdf/10.1073/pnas.2212199119"> pdf</a> ] </p>
<p> Choi H, Jung C, Kang T, Kim H, <font color="#FF7975">Kwak IY</font> (2022) Explainable Time-series Prediction using a Residual Network and Gradient-based Methods, <a href="https://ieeeaccess.ieee.org/"> IEEE Access</a>, 10, 108469 - 108482
<br>
  [ <a href="https://ieeexplore.ieee.org/document/9916238">IEEE Xplore</a> | <a href="https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=9916238"> pdf</a> ] </p>
<p> Park YB, Kim H, Lee HJ, Baek SH, <font color="#FF7975">Kwak IY</font>, Kim SH (2022) The Clinical Application of Machine Learning Models for Risk Analysis of Ramp Lesions in Anterior Cruciate Ligament Injuries, <a href="https://journals.sagepub.com/home/AJS"> The American Journal of Sports Medicine</a>, online first.
<br>
  [ <a href="https://pubmed.ncbi.nlm.nih.gov/36412925/">pubmed</a> | <a href="kim2022"> pdf</a> ] </p>
<p> <font color="#FF7975">Kwak IY</font>, Kim KS, Min HJ (2022) Differential effect of meteorological factors and particulate matter with ≤ 10-µm diameter on epistaxis in younger and older children, <a href="https://www.nature.com/srep/"> Scientific Reports</a>, 12, 21029.
<br>
  [ <a href="https://www.nature.com/articles/s41598-022-25630-3.pdf"> pdf</a> ] </p>



  <h4 id="2021">2021</h4>
<p> <font color="#FF7975">Kwak IY</font>, Kwag S, Lee J, Huh JH, Lee
  CH, Jeon Y, Hwang J, Yoon J (2021) ResMax: Detecting Voice Spoofing
  Attacks with Residual Network and Max Feature
  Map <a href="https://www.micc.unifi.it/icpr2020/">The
  25th International Conference on Pattern Recognition (ICPR 2020)</a>, Milan, Italy, 2021, 4837-4844 <br>
    [ <a href="kwak2021_icpr.pdf">pdf</a> ] </p>
<p> Lee Y, Lim S, <font color="#FF7975">Kwak IY</font> (2021) CNN-Based Acoustic Scene Classification System, <a href="https://www.mdpi.com/journal/electronics"> Electronics</a>, 10(4), 371,
<br>
  [<a href="https://www.mdpi.com/2079-9292/10/4/371">Link</a> | <a href="https://www.mdpi.com/2079-9292/10/4/371/pdf">pdf</a>] </p>

<p> Kim K, <font color="#FF7975">Kwak IY</font>, Min H (2021) Particulate Matter 10 (PM10) Is Associated with Epistaxis in Children and Adults, <a href="https://www.mdpi.com/journal/ijerph"> International Journal of Environmental Research and Public Health</a>, 18(9), 4809,
<br>
  [<a href="https://pubmed.ncbi.nlm.nih.gov/33946392/">PubMed</a> | <a href="https://www.mdpi.com/1660-4601/18/9/4809/pdf">pdf</a>] </p>

<p> Gong W, Granados A, Hu J, Jones M, Raz O, Martinez IS, Zhang H, Chow KK, <font color="#FF7975">Kwak IY</font>, Retkute R, Prusokas A, Prusokas A, Khodaverdian A, Zhang R, Wang R, Rao S, Rennert P, Saipradeep V, Naveen S, Joseph T, Rao A, Srinivasan R, Peng J, Han L, Shang X, Garry DJ, Yu T, Chung V, Mason M, Liu Z, Guan Y, Yosef N, Shendure J, Telford M, Shapiro E, Elowitz MB, Meyer P. (2021) Benchmarked approaches for cell lineage reconstructions of in vitro dividing cells and in silico models of Caenorhabditis elegans and Mus musculus developmental trees, <a href="https://www.cell.com/cell-systems/home"> Cell Systems</a>, 12, 1-17.
<br>
[<a href="https://pubmed.ncbi.nlm.nih.gov/34146472/">PubMed</a> | <a href="https://github.com/ikwak2/DCLEAR">R/DCLEAR software</a> | <a href="https://www.sciencedirect.com/sdfe/reader/pii/S2405471221001940/pdf"> pdf</a>]  </p>

<p> Park J, <font color="#FF7975">Kwak IY</font>, Lim C (2021) A Deep Learning Model with Self-Supervised Learning and Attention Mechanism for COVID-19 Diagnosis Using Chest X-ray Images, <a href="https://www.mdpi.com/journal/electronics"> Electronics</a>, 10(16), 1996,
<br>
  [<a href="https://www.mdpi.com/2079-9292/10/16/1996">Link</a> | <a href="https://www.mdpi.com/2079-9292/10/16/1996/pdf">pdf</a>] </p>

<p> Won H, Kim B, <font color="#FF7975">Kwak IY</font>, Lim C (2021) Transfer Learning followed by Transformer for Automated Audio Captioning, <a href="http://dcase.community/workshop2021/proceedings"> Proceedings of the 6th Detection and Classification of Acoustic Scenes and Events 2021 Workshop (DCASE2021) </a>, Online.
<br>
  [<a href="https://www.youtube.com/watch?v=cVMBgBQUTh0"> Video </a> | <a href="http://dcase.community/documents/workshop2021/proceedings/DCASE2021Workshop_Kim_70.pdf">pdf</a>] </p>




  
<h4 id="2020">2020</h4>

<p> Yoon HY, <font color="#FF7975">Kwak IY</font> (2020) Association Modeling on Keyword and Abstract Data in Korean Port Research, <a href="http://ktra.ccpshost.com/eng/main.html"> Journal of Korea Trade</a>, 25 (5), 71-86
<br>
  [<a href="http://scholar.dkyobobook.co.kr/searchDetail.laf?barcode=4010027908940">Link</a> | <a href="yoon_jkt2020.pdf">pdf</a>] </p>

<p> Ahmed E, <font color="#FF7975">Kwak IY</font>, Huh JH, Kim I, Oh T, Kim H (2020) Void: A fast and light voice liveness detection system. <a href="https://www.usenix.org/conference/usenixsecurity20">The 29th USENIX Security Symposium (USENIX Security)</a>, Boston, USA, 2020 <br>
[<a href="https://www.usenix.org/conference/usenixsecurity20/presentation/ahmed">Link</a> | <a href="ahmed2020.pdf">pdf</a>] </p>


<h4 id="2019">2019</h4>
<p><font color="#FF7975">Kwak IY</font>, Huh J, Kim I, Han S, Yoon J. (2019) Voice Presentation Attack Detection through Text Converted Voice Command Analysis, <a href="https://chi2019.acm.org/">Proceedings of the 2019 CHI Conference on Human Factors in Computing Systems (CHI 2019) </a> Paper No. 598. Glassgow, UK, 2020<br> 
[<a href="https://dl.acm.org/citation.cfm?id=3300828">Digital Library</a> | <a href="kwak2019.pdf">pdf</a>] </p>

<p>Magli A, Baik J, Pota P, Cordero CO, <font color="#FF7975">Kwak IY</font>, Garry DJ, Love PE, Dynlacht BD, Perlingeiro R (2019)  Pax3 cooperates with Ldb1 to direct local chromosome architecture during myogenic lineage specification, <a href="https://www.nature.com/ncomms/" >Nature Communications</a>, 10, 2316<br>
[<a href="https://www.ncbi.nlm.nih.gov/pubmed/31127120">PubMed</a> | <a href="Magi_2019_2.pdf"> pdf</a>] 
</p>

<p>Magli A, Baik J, Mills LJ, <font color="#FF7975">Kwak IY</font>, Dillon BS, Gonzalez RM, Stafford DA, Swanson SA, Stewart R, Thomson JA, Garry DJ, Dynlacht BD, Perlingeiro R (2019) Time-dependent Pax3-mediated chromatin remodeling and cooperation with Six4 and Tead2 specify the skeletal myogenic lineage in developing mesoderm, <a href="https://journals.plos.org/plosbiology/">PLoS biology</a>, 17 (2), e3000153<br>
[<a href="https://www.ncbi.nlm.nih.gov/pubmed/30807574">PubMed</a> | <a href="Magi_2019.pdf"> pdf</a>] 
</p>
  

  
<h4 id="2018">2018</h4>

<p>Gong W, <font color="#FF7975">Kwak IY</font>(co-1st author), Pota P, Koyano-Nakagawa N, Garry DJ. (2018) DrImpute: imputing dropout events in single cell RNA sequencing data, <a href="https://bmcbioinformatics.biomedcentral.com/">BMC Bioinformatics</a> 19:220 <br> [<a href="https://cran.r-project.org/web/packages/DrImpute/index.html">R/DrImpute software</a> | <a href="drimpute2018.pdf">pdf</a>]</p>
            
<p> Gong W, <font color="#FF7975">Kwak IY</font>, Koyano-Nakagawa N, Pan W, Garry DJ. (2018) TCM visualizes trajectories and cell populations from single cell data. <a href="https://www.nature.com/ncomms/" >Nature Communications</a>. 9(1), 2749<br> [<a href="https://github.com/gongx030/tcm">R/tcm software</a> | <a href="tcm2018.pdf">pdf</a>]</p>

    <h4 id="2017">2017</h4>

<p><font color="#FF7975">Kwak IY</font>, Pan W. (2017)
          Gene- and pathway-based association tests for multiple
          traits with GWAS summary statistics, <a href="http://bioinformatics.oxfordjournals.org/">Bioinformatics</a>, 33(1), 64-71<br> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/27592708">PubMed</a> | <a href="https://cran.r-project.org/web/packages/aSPU/index.html">R/aSPU software</a> | <a href="kwak2016bi.pdf">pdf</a>]</p>
     
            
<p> Chen Z, Zhu W, Bender I, Gong W, <font color="#FF7975">Kwak IY</font>, Yellamilli A, Hodges TJ, Nemoto N, Zhang J, Garry DJ, Berlo JH (2017) Pathologic Stimulus Determines Lineage Commitment of Cardiac C-kit+ Cells. <a href="https://www.ahajournals.org/journal/circ" >Circulation</a>, circulationaha. 117.030137<br> [<a href="jop2018.pdf">pdf</a>]</p>

    <h4 id="2016">2016</h4>
    
<p> Wei P, Cao Y, Zhang Y, <font color="#FF7975">Kwak IY</font>, Xu Z, Crosby J, Boerwinkle E, Pan W. (2016) On Robust Association Testing
          for Quantitative Traits and Rare Variants. <a href="http://g3journal.org/">G3</a> 6(12), 3941-3950<br>
[<a href="peng2016.pdf">pdf</a>]</p>
            
                      
<p><font color="#FF7975">Kwak IY</font>, Pan W. (2016)
           Adaptive gene- and pathway-trait association testing with GWAS summary statistics.  <a href="http://bioinformatics.oxfordjournals.org/">Bioinformatics</a>, 32(8):1178-1184 <br> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/26656570">PubMed</a> | <a href="https://cran.r-project.org/web/packages/aSPU/index.html">R/aSPU software</a> | <a href="kwak2016.pdf">pdf</a>]

    <h4 id="2015">2015</h4>

    
<p>Pan W, <font color="#FF7975">Kwak IY</font>, Wei P. (2015)
A powerful pathway-based adaptive test for genetic association with common or rare variants. <a href="http://www.cell.com/AJHG/home">The American Journal of Human Genetics</a>, 97(1):86-98<br>
[<a href="http://www.ncbi.nlm.nih.gov/pubmed/26119817">PubMed</a> | <a href="https://cran.r-project.org/web/packages/aSPU/index.html">R/aSPU software</a> | <a href="pan2015.pdf"> pdf (1.5M)</a>]</p>

<p><font color="#FF7975">Kwak IY</font>, Moore CR, Spalding EP, Broman KW. (2015)
Mapping quantitative trait loci underlying function-valued traits using functional principal component analysis and multi-trait mapping. <a href="http://g3journal.org/">G3</a>, 6(1):79-86<br>
[<a href="http://www.ncbi.nlm.nih.gov/pubmed/26530421">PubMed</a> | <a href="https://github.com/ikwak2/funqtl">R/funqtl software</a> | <a href="ilyoup2016.pdf">pdf</a> ]</p>

    <h4 id="2014">2014</h4>

  
<p><font color="#FF7975">Kwak IY</font>, Moore CR, Spalding EP,Broman KW. (2014)
A simple regression-based method to map quantitative trait loci underlying
  function-valued phenotypes. <a href="http://www.genetics.org">Genetics</a>, 197(4):1409-1416<br>
[<a href="http://www.ncbi.nlm.nih.gov/pubmed/24931408">PubMed</a> | <a href="https://github.com/ikwak2/funqtl">R/funqtl software</a> 
| <a href="kwak2014.pdf"> pdf (1.2M)</a>]</p>

    <h4 id="2013">2013</h4>


<p>Moore CR, Johnson LS, <font color="#FF7975">Kwak IY</font>, Livny M, Broman KW, Spalding EP (2013) High-throughput computer vision introduces the time axis to a quantitative trait map of a plant growth response.<a href="http://www.genetics.org"> Genetics</a> 195(3):1077-1086<br>
[<a href="http://www.ncbi.nlm.nih.gov/pubmed/23979570">PubMed</a>
| <a href="spalding.pdf"> pdf (1.6M)</a>]</p>


    <h4 id="2010">2010</h4>


<p>Billings T, Sargent EE, Szatkiewicz JP, Leahy N, <font color="#FF7975">Kwak IY</font>, Bektassova N, Walker M, Hassold T, Graber JH, Broman KW, Petkov PM (2010)  Patterns of recombination activity on mouse chromosome 11 revealed by high resolution mapping.  <a href="http://www.plosone.org">PLoS One</a> 5:e15340<br>
[<a href="http://www.ncbi.nlm.nih.gov/pubmed/21170346">PubMed</a> | <a href="petko_chr11.pdf">pdf (483k)</a>]</p>

    <h4 id="2008">2008</h4>

<p><font color="#FF7975">Kwak IY</font> and 
      Huh MH (2008) Parallel Coordinate Plots of Mixed-Type
Data. <a href="http://www.kss.or.kr/journal/commu/commu_01.aspx">The
      Korean Communications in Statistics</a> 15,
      587 - 595 <br>
[<a href="https://github.com/ikwak2/APCP">R/APCP software</a> | <a href="pp2.pdf">pdf (578k)</a>]</p>

<p><font color="#FF7975">Kwak IY</font> and 
      Huh MH (2008) Andrews’ Plots for Extended
      Uses. <a href="http://www.kss.or.kr/journal/commu/commu_01.aspx">The
      Korean Communications in Statistics</a> 15,
      87 - 94 <br>
[<a href="https://github.com/ikwak2/APCP">R/APCP software</a> | <a href="pp1.pdf">pdf (479k)</a>]</p>

</table>


<p class=noindent>
<hr size = "7" noshade="noshade" />
<p class=noindent>
<table cellspacing="0" cellpadding="0" width=100%>
<tr><td align="left"><p class=noindent>
<a href="../index.html">Back to home page</a>

  </body>
</html>
