<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
          "http://www.w3.org/TR/html1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

  <head>
    <title> Publications </title>
    <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
    <link rel="stylesheet" type="text/css" href="../iystyle2.css" />

  </head>

    <h1> Publications </h1>

<hr size = "7" noshade="noshade" />

    <table>

<tr><td width=100><h4>2016</h4>

    <tr><td><td> Wei P, Cao Y, Zhang Y, <font color="#FF7975">Kwak IY</font>, Xu Z, Crosby J, Boerwinkle E, Pan W. (2016) On Robust Association Testing
          for Quantitative Traits and Rare Variants. <a href="http://g3journal.org/">G3</a>, Accepted for publication. 
          
    <tr><td><td><font color="#FF7975">Kwak IY</font>, Pan W. (2016)
          Gene- and pathway-based association tests for multiple
          traits with GWAS summary statistics, <a href="http://bioinformatics.oxfordjournals.org/">Bioinformatics</a>, DOI: 10.1093/bioinformatics/btw577 [<a href="http://www.ncbi.nlm.nih.gov/pubmed/27592708">PubMed</a> | <a href="https://cran.r-project.org/web/packages/aSPU/index.html">R/aSPU software</a> | <a href="kwak2016bi.pdf">pdf</a> ]
          
            
     <tr><td><td><font color="#FF7975">Kwak IY</font>, Pan W. (2016)
           Adaptive gene- and pathway-trait association testing with GWAS summary statistics.  <a href="http://bioinformatics.oxfordjournals.org/">Bioinformatics</a>, 32(8):1178-1184 <br> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/26656570">PubMed</a> | <a href="https://cran.r-project.org/web/packages/aSPU/index.html">R/aSPU software</a> | <a href="kwak2016.pdf">pdf</a> ]

<tr><td><td><font color="#FF7975">Kwak IY</font>, Moore CR, Spalding EP, Broman KW. (2016)
Mapping quantitative trait loci underlying function-valued traits using functional principal component analysis and multi-trait mapping. <a href="http://g3journal.org/">G3</a>, 6(1):79-86<br>
[<a href="http://www.ncbi.nlm.nih.gov/pubmed/26530421">PubMed</a> | <a href="https://github.com/ikwak2/funqtl">R/funqtl software</a> | <a href="ilyoup2016.pdf">pdf</a> ]

<tr><td width=100><h4>2015</h4>

      <tr><td><td>Pan W, <font color="#FF7975">Kwak IY</font>, Wei P. (2015)
A powerful pathway-based adaptive test for genetic association with common or rare variants. <a href="http://www.cell.com/AJHG/home">The American Journal of Human Genetics</a>, 97(1):86-98<br>
[<a href="http://www.ncbi.nlm.nih.gov/pubmed/26119817">PubMed</a> | <a href="https://cran.r-project.org/web/packages/aSPU/index.html">R/aSPU software</a> | <a href="pan2015.pdf"> pdf (1.5M)</a>]

<tr><td width=100><h4>2014</h4>

<tr><td><td><font color="#FF7975">Kwak IY</font>, Moore CR, Spalding EP,Broman KW. (2014)
A simple regression-based method to map quantitative trait loci underlying
  function-valued phenotypes. <a href="http://www.genetics.org">Genetics</a>, 197(4):1409-1416<br>
[<a href="http://www.ncbi.nlm.nih.gov/pubmed/24931408">PubMed</a> | <a href="https://github.com/ikwak2/funqtl">R/funqtl software</a> 
| <a href="kwak2014.pdf"> pdf (1.2M)</a>]


<tr><td width=100><h4>2013</h4>


<tr><td><td>Moore CR, Johnson LS, <font color="#FF7975">Kwak IY</font>, Livny M, Broman KW, Spalding EP (2013) High-throughput computer vision introduces the time axis to a quantitative trait map of a plant growth response.<a href="http://www.genetics.org"> Genetics</a> 195(3):1077-1086<br>
[<a href="http://www.ncbi.nlm.nih.gov/pubmed/23979570">PubMed</a>
| <a href="spalding.pdf"> pdf (1.6M)</a>]



<br>


<tr><td width=100><h4>2010</h4>


<tr><td><td>Billings T, Sargent EE, Szatkiewicz JP, Leahy N, <font color="#FF7975">Kwak IY</font>, Bektassova N, Walker M, Hassold T, Graber JH, Broman KW, Petkov PM (2010)  Patterns of recombination activity on mouse chromosome 11 revealed by high resolution mapping.  <a href="http://www.plosone.org">PLoS One</a> 5:e15340<br>
[<a href="http://www.ncbi.nlm.nih.gov/pubmed/21170346">PubMed</a> | <a href="petko_chr11.pdf">pdf (483k)</a>]


<tr><td width=100><h4>2008</h4>

<tr><td><td> Park M, <font color="#FF7975">Kwak IY</font> and 
      Huh MH (2008) Applications of Parallel
Coordinate Plots for Visualizing Gene Expression
      Data. <a href="http://www.kss.or.kr/journal/app/app_01.aspx">The
      Korean Journal of Applied Statistics</a> 21(6),
      911 - 921 <br>

<tr><td><td><font color="#FF7975">Kwak IY</font> and 
      Huh MH (2008) Parallel Coordinate Plots of Mixed-Type
Data. <a href="http://www.kss.or.kr/journal/commu/commu_01.aspx">The
      Korean Communications in Statistics</a> 15,
      587 - 595 <br>
[<a href="https://github.com/ikwak2/APCP">R/APCP software</a> | <a href="pp2.pdf">pdf (578k)</a>]

<tr><td><td><font color="#FF7975">Kwak IY</font> and 
      Huh MH (2008) Andrewsâ€™ Plots for Extended
      Uses. <a href="http://www.kss.or.kr/journal/commu/commu_01.aspx">The
      Korean Communications in Statistics</a> 15,
      87 - 94 <br>
[<a href="https://github.com/ikwak2/APCP">R/APCP software</a> | <a href="pp1.pdf">pdf (479k)</a>]
<tr><tr><tr><tr><tr><tr><tr><tr><tr><tr><tr><tr><tr><tr>
</table>
<p class=noindent>
<hr size = "7" noshade="noshade" />
<p class=noindent>
<table cellspacing="0" cellpadding="0" width=100%>
<tr><td align="left"><p class=noindent>
<a href="../index.html">Back to home page</a>

  </body>
</html>
